openff/toolkit/topology/molecule.py

```diff
@@ -2444,6 +2444,26 @@ def assign_partial_charges(
         openff.toolkit.utils.toolkits.AmberToolsToolkitWrapper.assign_partial_charges
         openff.toolkit.utils.toolkits.BuiltInToolkitWrapper.assign_partial_charges
         """
+
+        # Raise a warning when users try to apply these charge methods to "large" molecules
+        WARN_LARGE_MOLECULES: Set[str] = {
+            "am1bcc",
+            "am1bccelf10",
+            "am1-mulliken",
+            "am1bccnosymspt",
+            "am1elf10",
+        }
```

Comment:
I'm pretty sure I hit everything here - Gasteiger, `mmff94`, etc ran in a few ms on my molecule - but I could have missed something.